src/test/ui/const-generics/defaults/complex-unord-param.min.stderr

```diff
@@ -2,7 +2,7 @@ error: type parameters must be declared prior to const parameters
   --> $DIR/complex-unord-param.rs:8:41
    |
 LL | struct NestedArrays<'a, const N: usize, A: 'a, const M: usize, T:'a =u32> {
-   |                    ---------------------^----------------------^--------- help: reorder the parameters: lifetimes, then types, then consts: `<'a, A: 'a, T: 'a, const N: usize, const M: usize>`
+   |                    ---------------------^----------------------^--------- help: reorder the parameters: lifetimes, then types, then consts: `<'a, A: 'a, T: 'a = u32, const N: usize, const M: usize>`
```

Comment:
It's a bit unfortunate that this suggestion is incorrect (at the moment type parameter defaults and const generics cannot be used together), but making this change will result in the correct diagnostic (altering the user of this fact), so I think it's okay for now.